install-impi.sh

```diff
@@ -0,0 +1,4 @@
+curl -O http://registrationcenter-download.intel.com/akdlm/irc_nas/tec/15553/aws_impi.sh
```

Comment:
Who uses this, maintains it?

Reply:
Intel. This is the script we tell customers to use in our public EFA docs.